zuul.d/pipelines.yaml

```diff
@@ -30,6 +30,38 @@
         status: 'failure'
       mysql:
 
+- pipeline:
+    name: check-no-comments
+    description: |
+      Newly uploaded patchsets enter this pipeline to receive an
+      initial +/-1 Verified vote without comments.
+    manager: independent
+    trigger:
+      github:
+        - event: pull_request
+          action:
+            - opened
+            - changed
+            - reopened
+        - event: pull_request
+          action: comment
+          comment: (?i)^\s*recheck\s*$
+    start:
+      github:
+        status: pending
+        comment: false
+    success:
+      github:
+        status: 'success'
+        comment: false
+        status_url=https://status.openlabtesting.org/builds?project={change.project}&change={change.number},{change.patchset}&pipeline={pipeline.name}
```

Comment:
status-url

Reply:
Yes, I move it to zuul.conf

https://zuul-ci.org/docs/zuul/admin/drivers/github.html#attr-pipeline.%3Creporter%3E.%3Cgithub%20source%3E.status-url